src/components/LessonCard/LessonCard.tsx

```diff
@@ -108,63 +76,192 @@ const Paragraph = styled(P)`
   line-height: 19px;
 `;
 
+const Hr = styled.hr`
+  margin: var(--spacing-x-large) 0 var(--spacing-x-large) 0;
+`;
+
 const StyledButton = styled(Button)``;
 
-type Props = {
+const selectMascot = (mascotType: MascotType, isResourcesCard: boolean, progress: Progress) => {
```

Comment:
This again ties back to my explanation in the overall comment - I wouldn't call this specifically as "isResourcesCard", but to be about what specific pieces that are being configurable, eg. show title or not, show progress indicator, show which supplementary buttons, etc.